src/axom/inlet/Table.hpp

```diff
@@ -231,6 +235,109 @@ class Table : public SchemaCreator, public std::enable_shared_from_this<Table>
    *****************************************************************************
    */
   bool required();
+
+  /*!
+   *****************************************************************************
+   * \brief Registers the function object that will verify this Table's contents
+   * during the verification stage.
+   * 
+   * \param [in] The function object that will be called by Table::verify().
+   *****************************************************************************
+  */
+  std::shared_ptr<Table> registerVerifier(std::function<bool()> lambda);
+
+   /*!
+   *****************************************************************************
+   * \brief This will be called by Inlet::verify to verify the contents of this
+   *  Table and all child Tables/Fields of this Table.
+   *****************************************************************************
+  */
+  bool verify();
+
+  /*!
+   *****************************************************************************
+   * \brief Return whether this Table has a child Field with the given name.
+   *
+   * \return Boolean value of whether this Table has the child Field.
+   *****************************************************************************
+   */
+  bool hasChildField(const std::string& fieldName);
+
+  /*!
+   *****************************************************************************
+   * \brief Return whether this Table has a child Table with the given name.
+   *
+   * \return Boolean value of whether this Table has the child Table.
+   *****************************************************************************
+   */
+  bool hasChildTable(const std::string& tableName);
+
+  /*!
+   *****************************************************************************
+   * \brief Return whether a Feld with the given name is present in this Table's
+   *  subtree.
+   *
+   * \return Boolean value of whether this Table's subtree contains this Field.
+   *****************************************************************************
+   */
+  bool hasField(const std::string& fieldName);
```

Comment:
Why is there a hasField vs hasChildField (same for Table)? what is a child vs a subtree?

Reply:
Also "Feld"